src/Gtk.jl

```diff
@@ -1,6 +1,13 @@
 # julia Gtk interface
 module Gtk
 
+# Import binary definitions
+using GTK3_jll, Glib_jll, gdk_pixbuf_jll, adwaita_icon_theme_jll, gdk_pixbuf_jll
```

Comment:
Is there an extra `gdk_pixbuf_jll`?

```suggestion
using GTK3_jll, Glib_jll, gdk_pixbuf_jll, adwaita_icon_theme_jll
```

Reply:
I really, really want to ensure that it is included.